.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -159,7 +159,11 @@ steps:
 
           # Else, internal build image repo, download from internal sonic-mgmt repo
           else
-            curl -u $(AZP_REPO_ACCESS_TOKEN) "${{ parameters.MGMT_URL }}&commitOrBranch=${{ parameters.MGMT_BRANCH }}&api-version=5.0-preview.1&path=.azure-pipelines%2Fpr_test_scripts.yaml" -o ./.azure-pipelines/pr_test_scripts.yaml
+            if [ -z "$(MSSONIC-TOKEN)" ]; then
```

Comment:
Directly getting the token probably won't work. It is a secret stored in variable group. Need to pass the token to task via environment variable. 

Document: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#secret-variables 

"Unlike a normal variable, they are not automatically decrypted into environment variables for scripts. You need to explicitly map secret variables."

Reply:
I have tested locally, before refer to this template, add the group that contains `MSSONIC-TOKEN` variable, and then can get it.
Actually, the usage just looks seems as `AZP_REPO_ACCESS_TOKEN`, both stored in library group, and import as needed. The difference is that `AZP_REPO_ACCESS_TOKEN` is a PAT and `MSSONIC-TOKEN` is maintained automaticly.